contracts/contracts/CAPE.sol

```diff
@@ -285,8 +285,8 @@ contract CAPE is RecordsMerkleTree, RootStore, AssetRegistry, ReentrancyGuard {
 
                 _publish(note.inputNullifier);
 
-                commitments.add(note.mintComm);
                 commitments.add(note.chgComm);
+                commitments.add(note.mintComm);
```

Comment:
Yes the` and_submit `version fails.

Reply:
The bug is in the contract so I'm not sure why it would fail without submitting to the contract. We did have the bug duplicated inside the function to collect the commitments. This was fixed in 10f8d6294d79c851e2044df2abc3068bd37104b2 and 83f6cf3426d6ca3d64b61154811c8cae700603ea